src/fileWordCopy.ts

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+import { getAllWords } from './fileWordCount';
+
+const fileName = process.argv[2];
+
+fs.readFile(fileName, 'utf8', (err: Error, data: string) => {
+  if (err) {
+    console.log(err);
+  } else {
+    writeToFile('result.txt', getAllWords(data));
+  }
+});
+
+const writeToFile = (fileName: string, data: string[]): void => {
+  fs.writeFile(fileName, data.join('\n'), err => {
+    if (err) {
+      console.log(err);
+    }
+    else {
```

Comment:
Para mantener la misma forma que en la línea 9

```suggestion
    } else {
```